modules/ServerAPI/src/server/routes/SetupRoutes.coffee

```diff
@@ -1,27 +1,23 @@
 exports.setupRoutes = (app, loginRoutes) ->
 	app.post '/api/setup/:typeOrKind', exports.setupTypeOrKind
 
-exports.setupTypeOrKind = (req, resp) ->
+exports.setupTypeOrKindInternal = (typeOrKind, roles, callback) ->
```

Comment:
These changes are to make setupTypeOrKind a useable function within node rather than just a request. I verified that the setup routes still work by editing a ConfJSON.coffee file and making sure new items get loaded to the database properly.